packages/frontend/src/retro/components/buttons/ToggleTimerDialogButton.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Alarm, SnoozeOutlined } from "@mui/icons-material";
+import { isModerator } from "../../../common/utils/participantsUtils";
+import { TimerDialog } from "../dialogs/TimerDialog";
+import { useRetroContext } from "../../context/RetroContext";
+import { useUserContext } from "../../../common/context/UserContext";
+import { useDialog } from "../../../common/hooks/useDialog";
+import { useTimer } from "../../hooks/useTimer";
+
+import { TimerStatus } from "../../types/retroTypes";
+import { WiggleActionButton } from "../../../common/components/buttons/WiggleActionButton";
+import useTimedEffect from "../../hooks/useTimedEffect";
+
+export function ToggleTimerDialogButton() {
+  const { isOpen, closeDialog, openDialog } = useDialog();
+  const { retroState, handleStopTimer } = useRetroContext();
+  const { timerStatus } = retroState;
+  const { user } = useUserContext();
+
+  const { minutes, seconds, remainingTimeLabel } = useTimer({
+    onTimerFinish: handleTimerFinish,
+  });
+  const { isEffectActive, startEffect } = useTimedEffect({ effectLength: 3000 });
+  function handleOpenDialog() {
```

Comment:
empty line

Reply:
Added it as rule, now eslint sets an empty line between const and functions